pkgs/applications/version-management/git-and-tools/git-reparent/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, makeWrapper, git, gnused }:
+
+stdenv.mkDerivation rec {
+  name = "git-reparent-${version}";
+  version = "2018-11-27";
```

Comment:
The commit itself is from 2017-09-03, so it would maybe be a better date to set as a version?

Reply:
sure, updated